crates/ark/src/modules/positron/options.R

```diff
@@ -36,3 +36,12 @@ options(plumber.docs.callback = function(url) {
 options(shiny.launch.browser = function(url) {
     .ps.ui.showUrl(url)
 })
+
+# Show Profvis output in the viewer
+options(profvis.print = function(x) {
+    # Render the HTML content to a temporary file
+    tmp_file <- htmltools::html_print(x, viewer = NULL)
+
+    # Pass the file to the viewer
```

Comment:
It is rather annoying that it doesn't resize. It means the window of text that you are reading is quite small to read.

It does resize in RStudio, I wonder what is different there?

https://github.com/user-attachments/assets/7c6e3e6d-66ae-43a1-b8a9-ae5407fe07a0